node-notes-cli/app.js

```diff
@@ -0,0 +1,91 @@
+const fs = require('fs');
+
+// Read
+if (process.argv[2].toString() === 'read') {
+  fs.readFile('data.json', 'utf-8', function read(err, data) {
+    if (err) {
+      throw err;
+    }
+    const content = JSON.parse(data);
+    for (var key in content.notes) {
+      console.log(`${key}: ${content.notes[key]}`);
+    }
+  });
+}
+
+// Create
+if (process.argv[2].toString() === 'create') {
+  fs.readFile('data.json', 'utf-8', function read(err, data) {
+    if (err) {
+      throw err;
+    }
+    const content = JSON.parse(data);
+    const thisKey = content.nextId;
+    content.nextId = thisKey + 1;
+    var value;
+    value = process.argv[3];
+    console.log('Added.');
+    Object.assign(content.notes, { [thisKey]: value });
+    const contentStri = JSON.stringify(content, null, 2);
+    fs.writeFileSync('data.json', contentStri);
```

Comment:
Synchronous writing is really slow, always use the asynchronous version (`writeFile`).